charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -139,27 +139,11 @@ ccd:
       enabled: true
     apiGatewayWeb:
       enabled: true
-    postgresql:
-      enabled: true
     s2s:
       enabled: false
     elastic:
       enabled: true
 
-  postgresql:
-    primary:
-      initdb:
-        scripts:
-          init.sql: |
-            CREATE DATABASE "data-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
```

Comment:
all the apps still need to be configured to use the secret you've created to auth to the dbs and to use the right db url etc

Reply:
sure - thanks -checking that now